Makefile

```
@@ -27,7 +27,7 @@ include release.mk
 
 
 bin/pytest: | bin/pip
-	bin/pip install pytest mock
+	bin/pip install pytest 'mock;python_version<"3.3"'
```

I'm fine with dropping Python 2.7 and, in fact, any Python < 3.6 support at this point, so this can be
```suggestion
	bin/pip install pytest
```